### ISSUE TITLE
Add URL parameters support

### ISSUE THREAD

**Issue by ludaavics** [collective/sphinxcontrib-httpexample#35]
Does `httpexample` support specifying URL parameters (other than by writing it in the URL)?

So instead of:

```reStructuredText
..  http:example:: python-requests curl

    GET /items?from=20170101&to=20171231&user_id=12&limit=20&sort=date(asc) HTTP/1.1
        Host: localhost
        Accept: application/json
        Authorization: Basic dXNlcjpwYXNzd29yZA==
```

Have something along the lines of:

```reStructuredText
..  http:example:: python-requests curl

        GET /items HTTP/1.1
        Host: localhost
        Accept: application/json
        Authorization: Basic dXNlcjpwYXNzd29yZA==

        :query from: 20170101
        :query to: 20171231
        :query user_id: 12
        :query limit: 20
        :query sort:date(asc)
```

Besides better readability, would be helpful when including `httpexample`s in python docstrings (e.g. for use with `sphinxcontrib.autohttp`) while respecting [PEP8](https://www.python.org/dev/peps/pep-0008/?#maximum-line-length).

I'm a newcomer to Sphinx, but happy to help where I can.

Thanks!

**Comment by datakurre** [comment]
There’s no such support yet, but I would not oppose it :)

**Comment by ludaavics** [comment]
Not sure this is the most elegant approach but took a shot at it. If looks good to you, will add to the doc, etc.

**Comment by datakurre** [comment]
Merged in 0.10.0. (Probably cut 1.0 release with the current feature set on some day.) Thank you for the contribution!
